Automattic/woocommerce-payments
# Allow issue of zero amount refunds

Fixes #7984

#### Changes proposed in this Pull Request

 - A special case for zero amount refunds with early exit.

Zero amount refunds can come handy in case merchant needs to re-stock items for an already refunded order.


#### Testing instructions

 - Create and complete payment for the order.
 - On the order page start a refund process:
   - Choose order items to be refunded and edit refunded amount and tax to be `0`
   - Select a "Re-stock refunded items" option.
 - Click on the "Refund $0.00 with WooPayments" button.
 - Process should complete without errors.
   - Re-stocked items should have a re-stock indicator.
   - There should be no note about $0.00 refund.

-------------------


**Post merge**
